packages/handlersjs-core/lib/handlers/pass-through.handler.ts

```diff
@@ -0,0 +1,34 @@
+import { from, Observable, of, throwError } from 'rxjs';
+import { mapTo } from 'rxjs/operators';
+import { HandlerArgumentError } from '../errors/handler-argument-error';
+import { Handler } from './handler';
+
+export class PassThroughHandler<T, S> extends Handler<T, S> {
+
+  constructor(public handler: Handler<T, S>) {
+
+    super();
+
+    if (!handler) { throw new HandlerArgumentError('Argument handler should be set.', handler); }
+
+  }
+
+  canHandle(input: T, intermediateOutput: S): Observable<boolean> {
+
+    return input && intermediateOutput ? of(true) : of(false);
```

Comment:
Even these input checks are not good? I just didn't see the point of a canHandle that always returns an Observable of true regardless of input?

Reply:
`canHandle` needs to indicate whether the input is acceptable for `handle`. If `handle` can do something with an undefined input, then `canHandle` should just let that happen. And since no generic handler knows whether it's nested handlers might just be happy with an undefined input, no generic handler should prevent undefined to be passed.

Reply:
Ok, got it.